src/github/interface.ts

```diff
@@ -41,6 +41,27 @@ export interface IAccount {
 	email?: string;
 }
 
+export interface ITeam {
+	name: string;
+	avatarUrl?: string;
+	url: string;
+	slug: string;
+	org: string;
+	id: string;
+}
+
+export function reviewerId(reviewer: ITeam | IAccount): string {
+	return (reviewer as ITeam).id || (reviewer as IAccount).login;
+}
+
+export function reviewerLabel(reviewer: ITeam | IAccount): string {
+	return (reviewer as ITeam).name || (reviewer as IAccount).login;
+}
+
+export function isTeam(reviewer: ITeam | IAccount): reviewer is ITeam {
+	return (reviewer as ITeam).id !== undefined;
```

Comment:
you could also check ``return "id" in reviewer;`` - then typescript helps you with the types.